src/include/com/amazonaws/kinesis/video/common/Include.h

```diff
@@ -44,80 +50,123 @@ extern "C" {
 #define STATUS_FILE_CREDENTIAL_PROVIDER_OPEN_FILE_FAILED                            STATUS_COMMON_PRODUCER_BASE + 0x00000022
 #define STATUS_FILE_CREDENTIAL_PROVIDER_INVALID_FILE_LENGTH                         STATUS_COMMON_PRODUCER_BASE + 0x00000023
 #define STATUS_FILE_CREDENTIAL_PROVIDER_INVALID_FILE_FORMAT                         STATUS_COMMON_PRODUCER_BASE + 0x00000024
-
-// Continue errors from the new common base
-#define STATUS_COMMON_BASE                                                          0x16000000
+/*!@} */
 
 ////////////////////////////////////////////////////
-// Main defines
+/// New common base status code

```

Comment:
Why "New"

Reply:
Oh. Added "New" based on this comment: https://github.com/awslabs/amazon-kinesis-video-streams-producer-c/blob/ee4eb03429f03737d9245e54f4d2fb13b08eaf80/src/include/com/amazonaws/kinesis/video/common/Include.h#L48